scripts/test_all_fuzzers.sh

```diff
@@ -23,9 +23,9 @@ do
         echo "[+] Skipping fmt and clippy for $fuzzer (--no-fmt specified)"
     fi
 
-    if [ -e ./Makefile ]; then
+    if [ -e ./build.toml ]; then
```

Comment:
build.toml sounds confusing if you are used to build.rs, is that a common name?
I somehow would have expected make.toml, what do other projects use?

Reply:
```
  --makefile <FILE>
            The optional toml file containing the tasks definitions [default: Makefile.toml]
```

Reply:
ok